content/docs/guides/components/tftraining.md

```diff
@@ -203,8 +204,34 @@ Typically you will want to change the following values
 
    * For example, you might need to configure various environment variables to talk to datastores like GCS or S3
 
-1. Attach PV's if you want to use PVs for storage.
+1. Attach PVs if you want to use PVs for storage.
 
+### Accessing the TFJob dashboard
+
+The TFJob dashboard is available at `<path>/tfjobs/ui/` (port `8080`).
+Specifically:
+
+* If you're using the central Kubeflow UI, you can access the TFJob dashboard
+  by clicking **TFJOB DASHBOARD**:
+
+    ![Central UI](/docs/images/central-ui.png)
```

Comment:
This doesn't show any image for me in preview

Reply:
Hmm, that's weird. I can see the image.